README.md

```diff
@@ -1 +1,40 @@
-# tutorial-linters
\ No newline at end of file
+# tutorial-linters
+# Build With
+ HTML
+ CSS
+# Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+  Setup
+  Install
+  Usage
+  Run tests
+  Deployment
+  Authors
+# 👤 Author1
+
+  GitHub: @githubhandle
+  Twitter: @twitterhandle
+  LinkedIn: LinkedIn
+# 👤 Author2
+
+  GitHub: @githubhandle
+  Twitter: @twitterhandle
+  LinkedIn: LinkedIn
```

Comment:
### OPTIONAL
- [ ] Kindly remove one author since this project is a solo 👍🏿 
- [ ] Please add your social media handles 🤝